### ISSUE TITLE
TOTP not working as expected

### ISSUE THREAD

**Issue by jamiep87** [TheJoeIaut/Bitwarden-Streamdeck-Plugin#2]
I am able to pull the email and password and it will enter it into the box i have the cursor in but when doing the same it just pastes the key that was used to setup the TOTP initially and not the code itself. Is this expected behavior?

**Comment by TheJoeIaut** [comment]
Yes you are right that is not the intended behaviour. Will be fixed in the next few days.

**Comment by TheJoeIaut** [comment]
Fixed in V2
